network/src/validator_network/state_synchronizer.rs

```diff
@@ -23,32 +25,33 @@ pub const STATE_SYNCHRONIZER_MSG_PROTOCOL: &[u8] = b"/libra/state-synchronizer/d
 /// `channel::Receiver<NetworkNotification>`.
 pub type StateSynchronizerEvents = NetworkEvents<StateSynchronizerMsg>;
 
+/// The interface from StateSynchronizer to Networking layer.
+///
+/// This is a thin wrapper around an `channel::Sender<NetworkRequest>`, so it is
```

Comment:
comment needs to be updated

Reply:
Ahh this was intended. I wanted to signal that this was in fact a wrapper around a `channel::Sender<NetworkRequest>` despite the `NetworkSender` indirection.

Reply:
Maybe just mention in the indirection in the comment? ..."This is a thin wrapper around NetworkSender, which in turn wraps channel::Sender<...?" for example.